src/backend/base/langflow/worker.py

```diff
@@ -31,10 +31,10 @@ def build_vertex(self, vertex: Vertex) -> Vertex:
 
 @celery_app.task(acks_late=True)
 def process_graph_cached_task(
-    data_graph: dict[str, Any],
-    inputs: dict | list[dict] | None = None,
-    clear_cache=False,
-    session_id=None,
+    data_graph: dict[str, Any],  # noqa: ARG001
```

Comment:
Not sure what the impact would be for celery to rmove these args

Reply:
celery is not supported atm (i.e we need to rebuild the integration)